R/uvhydrograph-render.R

```diff
@@ -219,78 +230,106 @@ createPrimaryPlot <- function(primaryData, primaryInfo, corrections, lims){
-                secondaryData[i][[1]]$time,
-                secondaryData[i][[1]]$value, 
-                secondaryInfo$plotStartDate, secondaryInfo$plotEndDate, 
-                secondaryInfo$secondary_lbl, secondaryInfo$tertiary_lbl, lims)
-            )
+            getSecondaryPlotConfig(list(estimated=secondarySeriesList$estimated), 
+                secondarySeriesList$estimated$time, secondarySeriesList$estimated$value,
+                startDate, endDate, 
+                uvInfo$label, tertiary_label, lims)
+        )
   }
   
   #corrected data
   plot_object <- 
       AddToGsplot(plot_object, 
           getSecondaryPlotConfig(list(corrected=secondarySeriesList$corrected), 
               secondarySeriesList$corrected$time, secondarySeriesList$corrected$value,
-              secondaryInfo$plotStartDate, secondaryInfo$plotEndDate, 
-              secondaryInfo$secondary_lbl, secondaryInfo$tertiary_lbl, lims)
+              startDate, endDate, 
+              uvInfo$label, tertiary_label, lims)
       )
+
+  #effective shift
+  if(!isEmptyVar(effective_shift_pts)){
+    plot_object <- 
+        AddToGsplot(plot_object, 
+            getSecondaryPlotConfig(list(effective_shift=effective_shift_pts), 
+                effective_shift_pts$time, effective_shift_pts$value,
+                startDate, endDate, 
+                uvInfo$label, tertiary_label, lims)
+        )
+  }
   
-  #estimated data
-  if(!isEmptyVar(secondarySeriesList$estimated)) {
+  if(!isEmptyVar(gage_height)){
     plot_object <- 
         AddToGsplot(plot_object, 
-            getSecondaryPlotConfig(list(estimated=secondarySeriesList$estimated), 
-                secondarySeriesList$estimated$time, secondarySeriesList$estimated$value,
-                secondaryInfo$plotStartDate, secondaryInfo$plotEndDate, 
-                secondaryInfo$secondary_lbl, secondaryInfo$tertiary_lbl, lims)
+            getSecondaryPlotConfig(list(gage_height=gage_height), 
+                gage_height$time, gage_height$value,
+                startDate, endDate, 
+                uvInfo$label, tertiary_label, lims)
         )
   }
   
-  #uncorrected data
-  if(!isEmptyVar(secondarySeriesList$uncorrected)) {
+  if(!isEmptyVar(meas_shift)){
     plot_object <- 
         AddToGsplot(plot_object, 
-            getSecondaryPlotConfig(list(uncorrected=secondarySeriesList$uncorrected),
-                secondarySeriesList$uncorrected$time, secondarySeriesList$uncorrected$value,
-                secondaryInfo$plotStartDate, secondaryInfo$plotEndDate, 
-                secondaryInfo$secondary_lbl, secondaryInfo$tertiary_lbl, lims)
+            getSecondaryPlotConfig(list(meas_shift=meas_shift), 
+                meas_shift$time, meas_shift$value,
+                startDate, endDate, 
+                uvInfo$label, tertiary_label, lims)
         )
   }
-
+  
+  if(!isEmptyVar(gw_level)){
+    plot_object <- 
+        AddToGsplot(plot_object, 
```

Comment:
can we update this function to not capitalize the first letter to match the rest of our style?